static/src/javascripts/boot.js

```diff
@@ -75,5 +75,21 @@ domready(() => {
             },
             'commercial'
         );
+    } else if (window.guardian.isEnhanced) {
```

Comment:
Would it not be better to prevent commercial from being requested if the switch is off? It saves the client from having to download and parse ~1MB of JavaScript